code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -411,12 +411,27 @@
 	// They stay locked down if their wire is cut.
 	if(wires?.is_cut(WIRE_LOCKDOWN))
 		state = TRUE
+	else if(!ai_lockdown)
+		lockdown_timer = addtimer(CALLBACK(src,PROC_REF(lockdown_override), FALSE), 10 MINUTES, TIMER_UNIQUE | TIMER_OVERRIDE | TIMER_DELETE_ME | TIMER_STOPPABLE)
+		to_chat(src, "<br><br>[span_alert("ALERT - Remote system lockdown engaged. Trying to hack the lockdown subsystem...")]<br>")
 	if(state)
 		throw_alert(ALERT_HACKED, /atom/movable/screen/alert/locked)
 	else
+		deltimer(lockdown_timer)
 		clear_alert(ALERT_HACKED)
 	set_lockcharge(state)
 
+/// Allows the borg to unlock themselves after a lenghty period of time.
+/mob/living/silicon/robot/proc/lockdown_override()
+	if(ai_lockdown)
+		to_chat(src, "<br><br>[span_alert("ALERT - Remote system lockdown override failed.")]<br>")
+		return
+		set_lockcharge(FALSE)
+		to_chat(src, "<br><br>[span_notice("ALERT - Remote system lockdown override successful.")]<br>")
+		if(connected_ai)
+			to_chat(connected_ai, "<br><br>[span_notice("ALERT - Cyborg [name] succesfully overriden the lockdown system")]<br>")
+	else
+		to_chat(src, "<br><br>[span_alert("ALERT - Remote system lockdown override failed.")]<br>")
```

Comment:
Proc seems to have got a little scrongled asa result of review process. Probably meant to look like this?
```suggestion
	if(ai_lockdown)
		to_chat(src, "<br><br>[span_alert("ALERT - Remote system lockdown override failed.")]<br>")
		return
	set_lockcharge(FALSE)
	to_chat(src, "<br><br>[span_notice("ALERT - Remote system lockdown override successful.")]<br>")
	if(connected_ai)
		to_chat(connected_ai, "<br><br>[span_notice("ALERT - Cyborg [name] succesfully overriden the lockdown system")]<br>")
```

Reply:
What the fuck happened here... Yeah, I think tabs went wrong